sourcemap-uploader/src/index.ts

```diff
@@ -202,6 +202,6 @@ program
   )
   .action(uploadSourcemaps);
 
-if (require.main === module) {
+if (process?.mainModule?.filename === __filename) {
```

Comment:
guessing this is one of the merge conflicts - the conditional shouldn't be there anymore!

Reply:
yup, my bad!